v4_MIGRATION_GUIDE.md

```diff
@@ -411,9 +411,8 @@ public class PayPalActivity extends AppCompatActivity {
   }
 
   private void myRequestBillingAgreementMethod() {
-    PayPalRequest request = new PayPalRequest()
-        .localeCode("US")
-        .billingAgreementDescription("Your agreement description");
+    PayPalVaultRequest request = new PayPalVaultRequest();
+    request.setBillingAgreementDescription("Your agreement description");
 
     payPalClient.requestBillingAgreement(this, request, (error) -> {
```

Comment:
I think it would make sense to update these code snippets to use `tokenizePayPalAccount` instead of the deprecated methods.

Reply:
Agreed - updated.